life_expectancy/tests/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""Test suite for the 1st assignment"""
```

Comment:
__init__ files are not made to store constants, please place this in config.py instead